libc-test/semver/openbsd.txt

```diff
@@ -781,7 +798,26 @@ SIGEMT
 SIGINFO
 SIGIO
 SIGSTKSZ
+SIOCADDMULTI
+SIOCAIFADDR
+SIOCAIFADDR_IN6
+SIOCDELMULTI
+SIOCDIFADDR
+SIOCDIFADDR_IN6
 SIOCGIFADDR
+SIOCGIFBRDADDR
+SIOCGIFDSTADDR
+SIOCGIFFLAGS
+SIOCGIFMTU
```

Comment:
`SIOCGIFMTU` isn't defined (whereas `SIOCSIFMTU` is defined in `src/unix/bsd/netbsdlike/mod.rs`)

Reply:
Added `SIOCGIFMTU` definition to `src/unix/bsd/netbsdlike/mod.rs`